site/gatsby-site/src/components/landing/QuickSearch.js

```diff
@@ -14,11 +14,11 @@ export default function QuickSearch() {
     const width = window.innerWidth;
 
     if (width >= 350 && width < 450) {
-      setSearchPlaceholder(t('Search 2000+ reports'));
+      setSearchPlaceholder(t('Search 2500+ reports'));
```

Comment:
@cesarvarela this wasn't translating so I added keys to the locales. However, it is still not translating. Is `landing.json` not the proper locale for QuickSearch?

Reply:
If you look at a few lines below, it is fetching translations from any of these three:
```
  const { t } = useTranslation(['translation', 'landing', 'actions']);
```
It should work 🤔 though I don't like that it is being used before being declared.

Reply:
Hmm, you are thinking reordering the declaration might fix it?

Reply:
yes